src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ACTAccessRequirementWidget.java

```diff
@@ -77,36 +97,63 @@ public void onSuccess(String rootWikiId) {
 		deleteAccessRequirementButton.configure(ar);
 		manageAccessButton.configure(ar);
 		subjectsWidget.configure(ar.getSubjectIds(), true);
-		refreshApprovalState();
+		lazyLoadHelper.setIsConfigured();
 	}
 	
-	public void refreshApprovalState() {
-		//TODO:  set up view based on DataAccessSubmission state
+	public void setDataAccessSubmissionStatus(ACTAccessRequirementStatus status) {
+		submissionId = status.getSubmissionId();
 		view.resetState();
-		// (for testing)
-		// if no request has been submitted:
-		view.showUnapprovedHeading();
-		view.showRequestAccessButton();
-		// else if a request has been submitted on your behalf
-//		view.showUnapprovedHeading();
-//		view.showRequestSubmittedMessage();
-		// else if you submitted a request
-//		view.showUnapprovedHeading();
-//		view.showRequestSubmittedMessage();
-//		view.showCancelRequestButton();
-		// else if your request submission has been approved
-//		view.showApprovedHeading();
-//		view.showRequestApprovedMessage();
-//		view.showUpdateRequestButton();
-		// else if your submitted request was rejected
-//		view.showUnapprovedHeading();
-//		view.showRequestRejectedMessage(reason);
-//		view.showUpdateRequestButton();
+		switch (status.getState()) {
+			case SUBMITTED:
+				// TODO: request has been submitted on your behalf, or by you?
```

Comment:
If accessor (and not the submitter), then show the submitter user badge, but do not show request.